stixcore/processing/publish.py

```diff
@@ -349,6 +352,116 @@ def send_mail_report(files):
             logger.error(f"Error: unable to send report email: {e}")
 
 
+def update_ephemeris_headers(fits_file, spice):
+    product = Product(fits_file)
+    if product.level in ['L1', 'L2']:
+        ephemeris_headers = \
+            spice.get_fits_headers(start_time=product.utc_timerange.start,
+                                   average_time=product.utc_timerange.center)
+        with fits.open(fits_file, 'update') as f:
+            for hdu in f:
+                now = datetime.now().isoformat(timespec='milliseconds')
+                hdu.header['HISTORY'] = f'updated ephemeris header with latest kernel at {now}'
+                hdu.header.update(ephemeris_headers)
+                # just update the first prima HDU
+                break
+        logger.info(f"updated ephemeris headers of {fits_file}")
+
+
+def addBSDComment(p, rid_lut):
```

Comment:
```suggestion
def add_BSD_comment(p, rid_lut):
```
Also add short docstring, I guess from the context rid is request but not sure